src/components/modules/toolbar/inline.ts

```diff
@@ -170,7 +170,7 @@ export default class InlineToolbar extends Module {
   /**
    * Shows Inline Toolbar
    */
-  private open(): void {
```

Comment:
@neSpecc This file is also named `inline.ts`. Should we rename this as well?

Reply:
no, it's included at the `/toolbar/` directory, so this is a redundant addition